### DISCUSSION TITLE
Help with regards to upgrading LTSP?

### DISCUSSION THREAD

**Discussion by JB-Prof** [ltsp/ltsp#866] [Q&A]
Hello.

We have been running LTSP for many years and I'm looking for advice on how best to proceed with upgrading please.

Our current server is a [HP ProLiant DL360 G5](https://support.hpe.com/hpesc/public/docDisplay?cc=us&docId=emr_na-c00715072&lang=en-us) with 2 x Dual Core Xeon 5160 @ 3.00 Ghz, 16 GB RAM, 300 GB RAID 5 SAS drives, set up in a dual-NIC old LTSP setup with one NIC for the Internet and the other NIC for an isolated LTSP thin-client gigabit network.

The thin clients we are using are Intel NUCs [BOXNUC6I5SYH I5-6260U](https://ark.intel.com/content/www/us/en/ark/products/89190/intel-nuc-kit-nuc6i5syh.html) computers with 4 GB RAM each.

The server is running Ubuntu 16.04 LTS which is past its EOL so we're in need of upgrading.

A new requirement for our users is to be able to run a certain Android app from their desktop.  I've had good success with Anbox, but checking their website indicates they only support up to Ubuntu 20.04.  

So my question is, if do a clean install of Ubuntu (probably Ubuntu Mate, but any spin is fine) of 20.04 and then install the current, up-to-date version of LTSP, will my server and clients be able to support being thick clients?  Or can I set it up so (for example) Anbox runs on the server in the old LTSP way, while the rest of the interface runs on the clients only, or some other permutation?  Or should I leave it as the old LTSP system?

Any advice is greatly appreciated!
Thank you.

**Comment by alkisg** [comment]
AFAIK anbox also supports 22.04. It would be a good idea to do a test installation of e.g. Ubuntu MATE 22.04 and anbox somewhere and see if you can get everything working the way you like.

The new LTSP supports both "fat/thick clients" out of the box, and "remoteapps" that run on the server via an ltsp.conf directive. So it should be able to do whatever the old LTSP did.

Your hardware seems like it should be able to run 22.04 without issues.

**Reply by JB-Prof** [reply]
Hello @alkisg.

Thank you for the reply.

When you say I should do a test installation of Ubuntu & Anbox, are you recommending I do this on the end-user NUC clients or server + client?

According to their website, [they officially support up to 20.04](https://github.com/anbox/anbox#supported-linux-distributions) with EOL of April, 2025,	so I'm not sure if there is another alternative which allows for running an Android app on LTSP clients?  We only need to run one app, not a full Android environment.

Thank you so much.

**Comment by alkisg** [comment]
- Test installation on the end-user NUC (on in a VM)
- No idea what Anbox supports but in many cases the docs are just out of date
- There's also [WayDroid](https://waydro.id/) which seems more maintained; you'd probably need GNOME for that as it's Wayland-only
- All these above are completely unrelated to LTSP, so that's all the help I can offer on this topic :)
